model/qti/ImportService.php

```diff
@@ -425,7 +426,7 @@ public function importQtiItem(
         array $metadataInjectors = [],
         array $metadataGuardians = [],
         array $metadataClassLookups = [],
-        array $sharedFiles = [],
+        array &$sharedFiles = [],
```

Comment:
The behavior is getting even more unpredictable here :D

Reply:
You're perfectly right. I removed the default value.